### ISSUE TITLE
PDF Summary Plugin dosen't work

### ISSUE THREAD

**Issue by AutoAPEAI** [microsoft/TaskWeaver#202]
**Describe the bug**
After I uploaded a pdf file and asked the agent to summarize for me, the error pops says that it's not able to fetch any LLM config details.

The execution of the generated python code above has failed

During execution, the following messages were logged:
Traceback (most recent call last):

  Cell In[2], line 2
    summary, description = paper_summary("PhD_Thesis___Chaoyang_He___April_2022.pdf")

  File /var/folders/mv/km2zkg_514d6dzr1mb_j92y00000gn/T/tmp7bzg3y0b/paper_summary.py:23 in __call__

  File <frozen os>:684 in __setitem__

  File <frozen os>:758 in encode

TypeError: str expected, not NoneType

2024-02-09 16:30:07 - Use back up engine: True
2024-02-09 16:30:09 - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"

**To Reproduce**
1. Start the service using `chainlit run app.py`
2. Uploaded a pdf file
3. Type the user query "can you summarize the pdf for me"
4. Wait for the response
5. See error

**Expected behavior**
The expected result is that agent will generate a summary according to the pre-built prompt

**Screenshots**
<img width="796" alt="image" src="https://github.com/microsoft/TaskWeaver/assets/150793191/aaffde78-ba49-4076-a27f-946b1111da76">

**Environment Information (please complete the following information):**
 - OS: [mac Sonoma 14.3 (23D56)]
 - Python Version [3.11]
 - LLM that you're using: [GPT-4]
 - Other Configurations except the LLM api/key related: [None]

**Additional context**
None

**Comment by zhangxu0307** [comment]
Hi, @AutoAPEAI ! Did you configurate the llm parameters in this [YAML file](https://github.com/microsoft/TaskWeaver/blob/e76442eed7274bf97ca076e813afa9ac20d5002e/project/plugins/paper_summary.yaml#L25C1-L30C19)?

**Comment by AutoAPEAI** [comment]
> Hi, @AutoAPEAI ! Did you configurate the llm parameters in this [YAML file](https://github.com/microsoft/TaskWeaver/blob/e76442eed7274bf97ca076e813afa9ac20d5002e/project/plugins/paper_summary.yaml#L25C1-L30C19)?

Thx, it worked now!
